src/components/PoliceNewsThumbnail.jsx

```diff
@@ -1,10 +1,21 @@
 import React from "react";
 
+const dg_police_news_icon = {
+  padding: "50px 55px",
```

Comment:
I don't think this matches up with the spacing in the design system. This should use one of the variable values. @theilman7 should be able to identify these for you.

Reply:
@theilman7 I'll differ to your opinion but this was making the icon a rectangle because dg_card_content adds a 20px bottom margin which isnt needed here. So I added an additional style to the icon to remove the bottom margin and made the background 60px all around which is the largest padding and looks more in line with XD. Next lower is 45 but theres a lot more white space. So just let me know what you think